docs/source/index.rst

```diff
@@ -17,10 +20,14 @@ brain white matter connections.
   create and link to page that identifies the supported `tissue properties`
   (believe this includes both dti and dki? fa, md, cfa?, and pdd?)
 
+.. note::
 
-.. todo::
-  include detailed page that distguished tractometry versus tractography
-  and benefits
+  Tractography is concerned with the visualization and mapping of white matter
```

Comment:
I think that we should move this paragraph into a page of its own in the "explanations" section of the documentation under the title "What are tractography, tractometry and voxel-based analysis?" or something like that.

Reply:
I think that since this is quick explanation of the two, we can leave it here for now. We can fill out the explanations page with something more detailed later.